docs/changing-rules.md

```diff
@@ -30,27 +30,9 @@ http_archive(
 )
 ```
 
-The `@build_bazel_rules_nodejs` module distributed on GitHub releases is harder to work with.
-It has different content from the sources in this repo.
-This is true for packages published to `@bazel/` on npm as well.
-These packages have only runtime bazel dependencies, but the sources depend on development dependencies. For example, the `@bazel_skylib` library is a development-time transitive dependency, while an npm package would have that dependency statically linked in.
+## Patching the rules_nodejs release
 
-If you have a small change, it's easiest to just patch the distributed artifacts rather than build from source. However if you're doing active development in rules_nodejs or have a policy of not depending on release artifacts, it's possible to depend directly on sources. This is not yet documented; file an issue on our repo if you think you need this.
-
-## Patching the npm packages
-
-The pattern we use most commonly is to use [patch-package]. To store your local changes to the npm packages follow the steps:
-
-1. `npm i -D patch-package`
-1. Edit the target package in your `node_modules`
-1. Run `npx patch-package <npm package>`. This will store the patch in the `patches/` directory in the root of the workspace
-1. Add `"postinstall": "patch-package"` to the `package.json` in your repo to apply the patches when building dependencies (aka at `npm install`)
-
-[patch-package]: https://www.npmjs.com/package/patch-package
-
-## Patching the build_bazel_rules_nodejs release
-
-`build_bazel_rules_nodejs` has a distribution format which is a tgz published to GitHub, and this can make it tricky to make casual changes without forking the project and building your own release artifacts.
+`rules_nodejs` has a distribution format which is a tgz published to GitHub, and this can make it tricky to make casual changes without forking the project and building your own release artifacts.
```

Comment:
this is not true anymore since you can consume from source if you want to from any commit on the 6.x branch